__tests__/tests.test.js

```diff
@@ -233,3 +233,65 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("PATCH 200; responds with a 200 code and updates an existing article, and responds to user with the updated article", () => {
+    const update = {
+      inc_votes: -20,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(update)
+      .expect(200)
+      .then(({ body }) => {
+        let updatedArticle = body.update;
+        expect(updatedArticle).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
```

Comment:
I think it would be worth clearly asserting that votes have decreased here. You aren't proving the patch request is working :)